Command/HealthCheckCommand.php

```diff
@@ -58,6 +58,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $reporter = $this->getContainer()->get('liip_monitor.helper.raw_console_reporter');
         } else {
             $reporter = $this->getContainer()->get('liip_monitor.helper.console_reporter');
+            if (method_exists($reporter, 'suppressOKs')) {
```

Comment:
not sure if we need this check .. I guess technically someone could have extended the ``ConsoleReporter`` ..